src/iks003_incompressibleLinearElasticity.cpp

```diff
@@ -33,55 +33,37 @@
 
 using namespace Ikarus;
 using namespace Dune::Indices;
-template <typename Basis_, typename FERequirements_ = FErequirements<>, bool useEigenRef = false>
-struct Solid {
+template <typename Basis_, typename FERequirements_ = FErequirements<>, bool useEigenRef = false, bool useFlat = true>
+struct Solid : public FEBase<Basis_, useFlat, FERequirements_, useEigenRef> {
```

Comment:
Maybe for the purpose of this test it is not necessary right now to have all these templates. Solid could only be a template of the basis and FEBase<Basis_> and everything else as default. The other templates are for more advanced stuff we want to educate the users at another place.

Reply:
Here are my questions/thoughts that we can discuss:
1) Do we expect the users to go through the examples with the numbering system we provided (`iks001 ... iks010`) or can it be random? If it can be random, then such changes should be applied to all the examples similarly. If not, then how to decide from which example should the advanced stuff begin?
2) We should also think about who are the users and which message we want them to have as a take away. If someone wants to be a developer in the future and would like to start with learning the examples, then may be the template arguments are good to have as he/she can already know/have impression about the `class FEBase` may be. For a beginner, it is nice as `FEBase<Base_>`.
3) What will be the other place for the advanced stuffs?
In summary, I guess then my thoughts are basically to decide the purpose and scope of all the examples.